h/security/policy/_cookie.py

```diff
@@ -62,3 +79,6 @@ def vary_add(request, response):  # pylint:disable=unused-argument
             response.vary = list(vary)
 
         request.add_response_callback(vary_add)
+
+    def _get_cookie_value(self):
```

Comment:
This is only a helper method but in isolation is not clear that the values are (user_id, ticket_id)

Reply:
Sent a follow-up PR to add a docstring to clarify this: https://github.com/hypothesis/h/pull/8875